torch/testing/_internal/common_methods_invocations.py

```diff
@@ -678,6 +678,39 @@ def sample_inputs_index_select(op_info, device, dtype, requires_grad):
                         0, torch.tensor(0, dtype=torch.int64, device=device))),
             )
 
+# Missing to test the nondeterminism of the operation
+# https://github.com/pytorch/pytorch/issues/53352
+def sample_inputs_index_add(op_info, device, dtype, requires_grad):
+    # These testa are pretty much the same as those from index_copy.
+    # Perhaps merge?
+    def make_arg(shape, low=None, high=None, dtype=dtype, device=device, contiguous=True):
```

Comment:
Does this sample input generator need to wrap make_tensor() in make_arg()? It doesn't seem like make_arg() is doing anything here. In the test test_torch.py make_arg() at least modifies the tensor's shape before calling make_tensor().

Reply:
I just use it so that I do not need to pass `dtype`,  `device` , `requires_grad` and `contiguous` every time I call it, which makes the code unnecessarily verbose.

Reply:
`contiguous` has a default, but you're correct the other three do need to be set a lot (which was part of the point of getting people to adopt make_tensor(), since it's easy to forget passing dtype and device in the tests!).

If you really want to do this then partial the function:

```
my_tensor = partial(make_tensor, dtype=dtype, device=device, requires_grad=requires_grad)
```